datasets.yaml

```diff
@@ -79,3 +117,106 @@ datasets:
               columns:
                 DATE: 'date'
           path: noaa-weather-data-jfk-airport/jfk_weather_cleaned.csv
+  sentiment_compositions_lexicon:
+    "1.0.2":
+      name: IBM Debater® Sentiment Compositions Lexicon
+      published: 2019-10-01
+      homepage: https://developer.ibm.com/exchanges/data/all/sentiment-composition-lexicons/
+      download_url: https://dax-cdn.cdn.appdomain.cloud/dax-sentiment-composition-lexicons/1.0.2/sentiment-composition-lexicons.tar.gz
+      sha512sum: 550da933b668270d890a4e12160202671894c357927e52250500e97b78965d7f3d4506396f9f21f50683a2fc54a4af5922f7fdd0ac7f01e4b5a2f1068451a519
+      license: cc_by_sa_30
+      estimated_size: 10M
+      description: "A dataset on the sentiment of phrases from the interaction between its constituents."
+      subdatasets:
+        unigrams:
+          name: Unigrams Sentiment Lexicon
+          description: Unigrams with their sentiment score
+          format:
+            id: txt
```

Comment:
Is this actually csv? txt doesn't get the column options

Reply:
Yeah it's technically a txt file but it can be read as a csv with a `'  '` as the separator. Should I just change the format to csv?

Reply:
Gotcha. How about we merge this after we have addressed https://github.com/CODAIT/pydax/issues/48 ?

Reply:
Alternatively, if we are to merge now, could you add a comment line:

    # TODO: Replace txt with csv when PyDAX supports specifying CSV delimiters.

Reply:
Sure we can wait until that feature is implemented.